File: javascript/1143.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var valor = parseInt(input);
var [a, b, c] = [1, 1, 1];
for (let i = 1; i <= valor; i++){
    console.log(`${a} ${b} ${c}`);
    a++;
    b=Math.pow(a,2);
    c=Math.pow(a,3);
}

File: C/1143.c
#include <stdio.h>
#include <math.h>

int main(){
    int valor, a = 1, b = 1, c = 1, i;
    scanf("%d", &valor);
    
    for(i = 1; i <= valor; i++) {
        printf("%d %d %d\n", a, b, c);
        
        a++;
        b = pow(a, 2);
        c = pow(a, 3);
    }

    return 0;
}